### PR TITLE
Tentative extract token

### PR DESCRIPTION
**Pull request by suprasanity**

### CODE REVIEW

**Comment by suprasanity** [comment]
Please take a look at the updated endpoint and tell me what do you think about the solution I came up with

**Review by williammadie — CHANGES_REQUESTED** [review]
I saw that there is a possibility of creating a custom decorator @User which might be a good enhancement of what you've done (see full example here : https://docs.nestjs.com/custom-decorators)

I think the following tasks would be to :

- create the custom decorator @User
- add this decorator to endpoints currently taking a userId in parameter inside a DTO. 
- remove the old userId from the DTOs but keep these DTO (don't switch to @parameter even if there is a single attribute inside the DTO)

File: src/api/library/library.controller.ts
Comment (williammadie) [inline src/api/library/library.controller.ts:1]: Fix deletion of chapter update endpoint

**Review by williammadie — CHANGES_REQUESTED** [review]
Fix deletion of chapter update endpoint

**Comment by suprasanity** [comment]
should be fine by now

**Review by williammadie — APPROVED** [review]

**Pull request merged**